index.html

```diff
@@ -8874,6 +8902,24 @@ <h2 id="WaveShaperNode">
               attribute therefore have no effect: the attribute must be set
               again in order to change the curve.
             </p>
+            <p>
+              To set the <code>curve</code> attribute, execute these steps:
```

Comment:
I agree with @rtoy. Anyone else want to weigh in? If not, I'll remove the WaveShaper piece.